optimism/src/mips/keccak/mod.rs

```diff
@@ -0,0 +1,67 @@
+pub mod column;
+
+pub const ZKVM_KECCAK_COLS: usize = 1965 + 6;
+pub const RATE: usize = 1088;
+pub const RATE_IN_BYTES: usize = RATE / 8;
+pub const DIM: usize = 5;
+pub const QUARTERS: usize = 4;
+
+fn grid_20(x: usize, q: usize) -> usize {
+    q + QUARTERS * x
+}
+
+fn grid_80(i: usize, x: usize, q: usize) -> usize {
+    q + QUARTERS * (x + DIM * i)
+}
+
+fn grid_100(y: usize, x: usize, q: usize) -> usize {
```

Comment:
This function is the same as above, please don't name according to caller semantics.

Reply:
Yeah, it is the same but I wanted to have different indices names to facilitate readability of cells afterwards in the constraints. Meaning, `i` ranges `[0..4)` and `y` ranges `[0..5)`, so using the same parameter name could be confusing.